sourcemash/frontend/static/js/routers/app_router.js

```diff
@@ -26,6 +27,15 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
         this._swapView(feedsView);
     },
 
+    browseFeeds: function() {
+        var browseView = new Sourcemash.Views.BrowseView({
```

Comment:
I’m not sure. I like the idea of categories only have articles you subscribed to (plus 1 not). It also seems more natural to browse through feeds for something to subscribe to than our self-made categories that aren’t exactly perfect. It’d make more sense if we chose the initial category set (e.g. Tech, Politics), but not with our current structure. 